cmd/core-metadata/res/configuration.toml

```diff
@@ -1,7 +1,7 @@
 DBType = 'mongodb'
 MongoDatabaseName = 'metadata'
-MongoDBUserName = ''
-MongoDBPassword = ''
+MongoDBUserName = 'meta'
```

Comment:
Why is this needed? Why not leaving empty strings?

Reply:
I thought it's more convenient to leave some default values here. I can change it back.